logger/src/lib.rs

```diff
@@ -60,6 +60,18 @@ fn notify(signals: &[c_int]) -> Result<channel::Receiver<c_int>, Error> {
     Ok(r)
 }
 
+pub fn init_console(service_name: &str) {
```

Comment:
Is init_console not a part of init_config?
And i notices that many codes are same
Why use two init entrance?

Reply:
Would you please reply it next time. because i don't know whether you accept it or not if you don't show your opinion.

And we only accept resonable comment. so you can say no if you think the comment is wrong.

btw, nice job!

Reply:
Sorry for not responding. I will reply next time.